js/views/startScreen.js

```diff
@@ -25,6 +25,10 @@ var StartScreen = React.createClass({
     this.handleResize();
   },
 
+  componentWillReceiveProps: function() {
```

Comment:
@mistermendez handleResize() is also called from ResizeMixin() and this calls with this.handleResize(nextProps); So I modified accordingly in startscreen.js and pushed the code changes

  componentWillReceiveProps: function(nextProps) {
    if (nextProps.contentTree.description != this.props.contentTree.description) {
      this.handleResize(nextProps);
    }
  },

  handleResize: function(nextProps) {
    var description = nextProps ? nextProps.contentTree.description : this.props.contentTree.description;
    if (ReactDOM.findDOMNode(this.refs.description)){
      this.setState({
        descriptionText: Utils.truncateTextToWidth(ReactDOM.findDOMNode(this.refs.description), description)
      });
    }
  },